### ISSUE TITLE
Fixed random Seed does not create the same results

### ISSUE THREAD

**Issue by rs3dtouch** [chigozienri/VQGAN-CLIP-animations#28]
Using a fixed random seed does not produce the same result, even with an _initial_image_ as starting point. 

I guess there are different seed values in some sub routines - do you think it is possible to fix all these to make 100% reproducible results? 

My hope would be, that animation would get more stable then. But maybe I'm wrong and this would not help to tame the temporal incoherence...

**Comment by chigozienri** [comment]
Good catch, sorry for the delay. Unfortunately I don't think this will affect the consistency from one frame to the next, just fix results between runs. I would accept a PR that addressed this, but I expect it's deep in the guts of PyTorch somewhere, and the expected benefit doesn't seem big enough for me to devote time to right now (even though I agree that it would be nice to have a fix).

**Comment by chigozienri** [comment]
@rs3dtouch I have found something that I think I could build into a solution to make this reproducible, but it would take some work, and would more memory/reduce performance. I don't think it's worth it (and I am sure it won't affect the animation stability) but if you think it would be useful for reasons other than animation stability, I could work on a toggle for it.
